includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -4796,7 +4812,6 @@ class AMP_Allowed_Tags_Generated {
 					'disallowed_ancestor' => array(
 						'amp-story',
 					),
-					'mandatory_parent' => 'body',
```

Comment:
Interesting. No longer does `amp-sidebar` have to be directly a child of the `body` element. Now it is merely [recommended](https://amp.dev/documentation/components/amp-sidebar/):

<blockquote><code>&lt;amp-sidebar&gt;</code> is <em>recommended</em> to be be a direct child of the <code>&lt;body&gt;</code> to preserve a logical DOM order (for accessibility) as well as to avoid altering its behavior by a container element. Note that having an ancestor of <code>amp-sidebar</code> with a set <code>z-index</code> may cause the sidebar to appear below other elements (such as headers), breaking its functionality.</blockquote>